src/screens/Home.jsx

```diff
@@ -6,16 +6,26 @@ import {
   WatchlaterVideos,
   FeaturedVideos,
 } from "../components";
+import { Link } from "react-router-dom";
+import styles from "./Home.module.css";
 
 const Home = ({ cname }) => {
   return (
     <main className={cname}>
-      <FilterTags />
+      <FilterTags page={"homePage"} />
```

Comment:
Since this is a string you can avoid writing curly brace.

Reply:
Sure, will correct it in the next PR.